app/leave-app/backend/service.bal

```diff
@@ -0,0 +1,736 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import leave_service.authorization;
```

Comment:
```suggestion
// under the License.

import leave_service.authorization;
```
New line

Reply:
Some files we have the newline
https://github.com/wso2/people-ops-suite/blob/248accf0960f0230cdb5561eddc77d363bd24fd2/app/leave-app/backend/constants.bal#L16

Reply:
For the imports we did that. If a file starts with a comment, then a new line is added. Otherwise a new line is added.

Reply:
Acknowledged.